src/main/java/org/dbflute/jetty/JettyBoot.java

```diff
@@ -236,19 +237,35 @@ public JettyBoot go() { // public as parts, no wait
         }
         prepareServer();
         final URI uri = startServer();
-        info("Boot successful" + (development ? " as development" : "") + ": url -> " + uri);
+        loggingBootSuccessful(uri);
         browseOnDesktopIfNeeds(uri);
         return this;
     }
 
+    private void loggingBootSuccessful(URI uri) {
+        final String serverHost = getServerHost();
+        if (serverHost != null) {
+            info("Boot successful" + (development ? " as development" : "") + ": url -> " + uri);
```

Comment:
The message building logic is almost same as the other so extract it from if-statement.
(Only url making needs if-statement)